ailerons_tracker_backend/models/ind_model.py

```diff
@@ -0,0 +1,41 @@
+""" Individual and Context models """
+from ailerons_tracker_backend.clients.supabase_client import supabase
+
+
+class Individual:
+    """ Model for an individual """
+
+    def __init__(self, form, image_urls: list[str]):
+        self.name: str = form['indName']
+        self.sex: str = form['indSex']
+        self.pictures = image_urls
+
+    def upload(self):
+        """ Insert the object as a new row in table 'individual' """
+
+        data = supabase.upsert(self, 'individual')
+        return data
+
+
+class Context:
+    """ Model for an individual's tagging context """
+
+    def __init__(self, ind_id, form):
```

Comment:
Probably waaaay overkill but until we have a chat with the folks at Ailerons again, this is a prototype.